docs/usage/administration/admin_api/federation.md

```diff
@@ -108,7 +108,37 @@ A response body like the following is returned:
 }
 ```
 
+**Parameters**
+
+The following parameters should be set in the URL:
+
+- `destination` - Name of the remote server.
+
 **Response**
 
 The response fields are the same like in the `destinations` array in
 [List of destinations](#list-of-destinations) response.
+
+## Reset connection timeout
+
+This API resets the retry timing for a specific remote server and tries to connect
+the remote server again. It does not wait for the next `retry_interval`.
```

Comment:
> and tries to connect
> the remote server again

I think this is wrong. AFAICS writing `(None, 0, 0)` to the DB will allow future federation requests to be attempted, but there's no generic "queue" of requests that we've prevented due to previous requests failing.

Reply:
Ahh, you're quite right. Thank you for explaining, and pointing out `notify_remote_server_up` in particular.